model.py

```diff
@@ -411,7 +411,7 @@ def discriminator(self, image, reuse=False):
             h1 = lrelu(self.d_bns[0](conv2d(h0, self.df_dim*2, name='d_h1_conv'), self.is_training))
             h2 = lrelu(self.d_bns[1](conv2d(h1, self.df_dim*4, name='d_h2_conv'), self.is_training))
             h3 = lrelu(self.d_bns[2](conv2d(h2, self.df_dim*8, name='d_h3_conv'), self.is_training))
-            h4 = linear(tf.reshape(h3, [-1, 8192]), 1, 'd_h4_lin')
+            h4 = linear(tf.reshape(h3, [-1, 8192]), 1, 'd_h3_lin')
```

Comment:
This is named d_h3_lin in the current saved checkpoint, so needs to keep that name for it to load.

However, I do agree that d_h4_lin is the correct numbering!